pymeasure/instruments/keithley/keithley6221.py

```diff
@@ -224,6 +224,18 @@ def waveform_duration_set_infinity(self):
         values={True: 1, False: 0},
         map_values=True,
     )
+    waveform_phasemarker_phase = Instrument.control(
+        ":SOUR:WAVE:PMAR?", ":SOUR:WAVE:PMAR %s",
```

Comment:
```suggestion
        ":SOUR:WAVE:PMAR?", ":SOUR:WAVE:PMAR %g",
```
I guess this is a floating point number, so I propose `%g`, make sense ?